lib/campaign/CampaignState.cpp

```diff
@@ -81,30 +86,40 @@ ImagePath CampaignRegions::getNameFor(CampaignScenarioID which, int colorIndex,
 {
 	auto const & region = regions[which.getNum()];
 
-	static const std::string colors[2][8] =
+	static const std::string colors[3][8] =
```

Comment:
From SonarCloud:

Use "std::array" or "std::vector" instead of a C-style array.